pkg/core/attributes/element.go

```diff
@@ -22,6 +22,7 @@ const (
 	Geo
 	NoElement
 	Physical
+	Burning
```

Comment:
instead of adding `Burning` element make a condition `ele == "burning"` in `evalElement` (https://github.com/genshinsim/gcsim/blob/main/pkg/conditional/enemy.go) and use `IsBurning` (example: https://github.com/kengzzzz/gcsim/blob/main/internal/artifacts/unfinishedreverie/unfinishedreverie.go)